### DISCUSSION TITLE
problemas com UiKit

### DISCUSSION THREAD

**Discussion by ClaudineiMonteiro** [balta-io/7187#86] [Perguntas e Respostas] [answered]
git clone git@github.com:uikit/uikit.git
Cloning into 'uikit'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @ClaudineiMonteiro, por favor tente o seguinte comando:
```
git clone https://github.com/uikit/uikit.git
```
